sdk/core/Azure.Core/src/Pipeline/HttpPipeline.cs

```diff
@@ -126,18 +129,68 @@ public Response SendRequest(Request request, CancellationToken cancellationToken
         /// </example>
         public static IDisposable CreateClientRequestIdScope(string? clientRequestId)
         {
-            return ReadClientRequestIdPolicy.StartScope(clientRequestId);
+#pragma warning disable CS8604 // Possible null reference argument.
```

Comment:
Don't ignore it, it's a good error :)

CreateHttpMessagePropertiesScope takes a dictionary of non-nullable values and `CreateClientRequestIdScope` takes a nullable string.

Reply:
Message.Properties values are nullable as well.